edge/pkg/common/dbm/db.go

```diff
@@ -72,9 +73,22 @@ func InitDBManager() {
 	DBAccess.Using(dbName)
 }
 
+// Clean db file if it is neccessary
+func Clean() {
+	// Check if db file is existing
+	_, err := os.Stat(defaultDataSource)
```

Comment:
Can't we directly do `os.Remove()` and then handle the `*PathError`?

Reply:
Done, Thanks